README.md

```diff
@@ -80,13 +80,15 @@ To specify which patients the client should extract data for, the configuration
 
 To successfully post extracted resources to the ICAREdata infrastructure, you _must_ modify the `awsConfig` object to provide proper AWS credentials. This can be done after your keycloak account has been set up.
 
-Each extractor uses various methods to gather data and format that data into [mCODE](http://hl7.org/fhir/us/mcode/index.html) profiled resources. Extractors may require additional configuration items that can be specified in the configuration file. In order to run the ICARE Extraction Client, you will need extractors for the following resources:
+Each extractor uses various methods to gather data and format that data into [mCODE](http://hl7.org/fhir/us/mcode/index.html) profiled resources. The `observation` extractor formats data into a general [FHIR R4](http://hl7.org/fhir/R4) profile. Extractors may require additional configuration items that can be specified in the configuration file. In order to run the ICARE Extraction Client, you will need extractors for the following resources:
```

Comment:
DM1: We did this because there's no corresponding mCODE profile that we wanted to map to, right? We just needed a full FHIR Observation extractor so we had that in support of mCODE needing Observations elsewhere?

Reply:
DM1: Yep! The Observation profile is not explicitly expanded on in the mCODE spec, but several of the profiles in the mCODE spec that'll be processed by this extractor are based on the general FHIR Observation profile. Also, all of the Vital Signs profiles are general FHIR profiles and not mCODE specific.